hack/scripts/aws/setup_aws_accounts.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+
+command -v aws >/dev/null 2>&1 || { echo >&2 "Script requires aws but it's not installed.  Aborting."; exit 1; }
+
+usage() {
+    cat <<EOF
+    usage: $0 [ OPTION ]
+    Options
+    -a        Assigned AWS Account ID 1
+    -b        Assigned AWS Account ID 2
+    -u        AWS IAM user name (Required)
+EOF
+}
+
+if ( ! getopts ":a:b:u" opt); then
+    echo ""
+    echo "    $0 requires an argument!"
+    usage
+    exit 1
+fi
+
+
+while getopts ":a:b:u" opt; do
+    case $opt in
+        a)
+            AWS_ACCOUNT_ID_1="$OPTARG"
+            ;;
+        b)
+            AWS_ACCOUNT_ID_2="$OPTARG"
+            ;;
+        u)
+            AWS_USERNAME="$OPTARG"
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            usage
+            exit 1
+            ;;
+        :)
+            echo "$0 Requires an argument" >&2
+            usage
+            exit 1
+            ;;
+        esac
+    done
+
+
+if [ -z "$AWS_ACCOUNT_ID_1" ]; then
+	usage
+fi
+
+if [ -z "$AWS_ACCOUNT_ID_2" ]; then
+	usage
+fi
+
+if [ -z "$AWS_USERNAME" ]; then
+	usage
+fi
+
+check_var() {
+    if [[ -z "$2" ]]; then
+        echo "$1 is not defined. Quitting ..."
+        exit 1
+    else
+        echo "$1=$2"
+    fi
+}
+
+assume_aws_role() {
+    AWS_ASSUME_ROLE=$(aws sts assume-role --profile "$2" --role-arn arn:aws:iam::"$1":role/OrganizationAccountAccessRole --role-session-name aao-account-setup)
```

Comment:
this or some more meaningful name
```suggestion
     local account_1=$1
      local account_2=$2
    AWS_ASSUME_ROLE=$(aws sts assume-role --profile "$account_2" --role-arn arn:aws:iam::"$account_1":role/OrganizationAccountAccessRole --role-session-name aao-account-setup)
```

Reply:
applied, thanks :)